docs/testing.md

```diff
@@ -125,6 +126,9 @@ and has three fields:
   `"pass"`, defining the information needed to talk to the IPMI
```

Comment:
Never mind, I saw some inconsistency in `docs/network-drivers.md` and was wondering which format is correct. +1